.taskcluster.yml

```diff
@@ -10,302 +10,305 @@ tasks:
               revision: cab4565345d0d0effa66f18fe91335dd6d744031
           trustDomain: mobile
       in:
-          $if: 'tasks_for in ["github-pull-request", "github-push", "action", "cron"]'
```

Comment:
This `if` statement does not exist in neither android-components nor fenix. I think it dates back when https://github.com/mozilla-mobile/reference-browser/pull/1187/files#diff-ac0229d1171c0983b27003af4c7441a5L95 wasn't as guarded as it is now.